coursedata/texts/de.yaml

```diff
@@ -1,21 +1,21 @@
 ui:
-    level_title: "Stufe"
+    level_title: "Level"
```

Comment:
This seems to be quite a big change, is `stufe` not right here? (I don't know a lot of German, so cc @dpprdan)